src/fielddef.ts

```diff
@@ -60,6 +60,17 @@ export interface FieldDef {
   title?: string;
 }
 
+/* Checks if two FieldDef are identical */
+export function identical(f1: FieldDef, f2: FieldDef): boolean {
```

Comment:
This is not scalable way to implement this.   You can utilize `json-stable-stringify` to compare fieldDef and it will remain correct no matter how we modify fieldDef.